### PR TITLE
Add nova change to set the vdpa ifname in vm xml file in victoria

### PR DESCRIPTION
**Pull request by wmousa**
Set the vif_name to be the vif net device, as it will be target dev
for the vdpa ports in the xml file

### CODE REVIEW

File: openstack/victoria/ovs-kernel/nova.patch [hunk 1]
```diff
@@ -1,8 +1,34 @@
...
+ def _nova_to_osvif_vif_ovs(vif):
+-    vif_name = _get_vif_name(vif)
++    vif_name = ""
+     vnic_type = vif.get('vnic_type', model.VNIC_TYPE_NORMAL)
++    if vnic_type == model.VNIC_TYPE_VIRTIO_FORWARDER:
++        # set the vif_name to be the vif net device, as it will be target dev
++        # for the vdpa ports in the xml file
++        vif_name = pci_utils.get_ifname_by_pci_address(
++                vif["profile"]['pci_slot'])
++    else:
```
Comment (moshe010) [inline openstack/victoria/ovs-kernel/nova.patch:25]: why do we need else part?

**Review by moshe010 — COMMENTED** [review]
Also were is the update for MTU?

File: openstack/victoria/ovs-kernel/nova.patch (hunk 1 above)
Comment (wmousa) [reply openstack/victoria/ovs-kernel/nova.patch:25]: I don't want to break something else, it was set before
https://github.com/openstack/nova/blob/master/nova/network/os_vif_util.py#L329

**Comment by wmousa** [comment]
> Also were is the update for MTU?

it's on another PR, pushing it

**Comment by wmousa** [comment]
@moshe010

**Review by moshe010 — APPROVED** [review]

**Pull request merged**